notification/events.go

```diff
@@ -306,6 +306,8 @@ func sendNotifications(ctx context.Context, events models.Events) models.Events
 
 			if !isHealthReportable(notif.Events, previousHealth, currentHealth) {
 				ctx.Logger.V(6).Infof("skipping notification[%s] as health change is not reportable", notif.ID)
+				notificationContext.log.WaitForResolved()
```

Comment:
we already save a record on `notification_send_history` table.

This will ensure that we finalize that record when the notification is skipped due to wait for evaluation.

Otherwise, we currenly have records in notification_send_history table with empty status & duration = 0 .